app/pktgen-stats.c

```diff
@@ -533,14 +534,14 @@ pktgen_process_stats(struct rte_timer *tim __rte_unused, void *arg __rte_unused)
         rate = &info->rate_stats;
         prev = &info->prev_stats;
 
-        rate->ipackets  = curr->ipackets - prev->ipackets;
-        rate->opackets  = curr->opackets - prev->opackets;
-        rate->ibytes    = curr->ibytes - prev->ibytes;
-        rate->obytes    = curr->obytes - prev->obytes;
-        rate->ierrors   = curr->ierrors - prev->ierrors;
-        rate->oerrors   = curr->oerrors - prev->oerrors;
-        rate->imissed   = curr->imissed - prev->imissed;
-        rate->rx_nombuf = curr->rx_nombuf - prev->rx_nombuf;
+        rate->ipackets  = (curr->ipackets - prev->ipackets) * Billion / (double)elapsed_ns + 0.5;
+        rate->opackets  = (curr->opackets - prev->opackets) * Billion / (double)elapsed_ns + 0.5;
```

Comment:
Would it be better to create a define
#define ROUNDUP 0.5 
Or some define name?

Reply:
Does `ROUND_FACTOR` seem a good name to you? I would avoid `ROUNDUP` since it rounds to the closest integer, not always up

Reply:
Yes, I agree ROUND_FACTOR is good.